src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -4,7 +4,7 @@ import cats.effect.IO
 import org.scalajs.dom._
 import outwatch.dom.helpers.DomUtils
 import rxscalajs.Observer
-import snabbdom.{DataObject, VNodeProxy, h}
+import snabbdom.{DataObject, VNodeProxy, hFunction}
```

Comment:
Why this rename? Instead of importing `hFunction` one could just write `snabbdom.h( ... )` everywhere instead.

Reply:
I renamed it because I think `h` is a pretty poor name for it (makes sense in JS where it's used a lot directly and it should be short, but that's not the case here). Also, the trait was already called `hFunction` and having a trait and an object with the same name is common in Scala.js facades.

It's still possible to have `import snabbdom.{hFunction => h}` if somebody really likes the `h` name.